src/main/java/com/karankumar/booksapi/repository/BookRepository.java

```diff
@@ -26,4 +26,7 @@
     List<Book> findAllBooks();
 
     Book findBookByIsbn13(String isbn13);
+  
+   @Query("SELECT b FROM Book b where upper(b.title) = upper(?1)")
```

Comment:
Could you please remove this line? Instead, could you please rename the method to `findByTitleIgnoreCase`?